service/src/test/java/bio/terra/catalog/controller/DatasetApiControllerTest.java

```diff
@@ -41,8 +42,13 @@ class DatasetApiControllerTest {
 
   private static final String API = "/api/v1/datasets";
   private static final String API_ID = API + "/{id}";
-  private static final String METADATA = """
-      {"some": "metadata"}""";
+  private static final String METADATA =
+      """
+      {"files": [{"count": 1,"dcat:byteSize": 63354880,"dcat:mediaType": "tar"}]}""";
+
+  private static final Map<String, Object> METADATA_MAP =
+      Map.of(
+          "files", List.of(Map.of("count", 1, "dcat:byteSize", 63354880, "dcat:mediaType", "tar")));
```

Comment:
I would use `objectMapper` to do this rather than copying the `METADATA` definition. This should work, after moving it below line 61.
```suggestion
  private final Map<String, Object> METADATA_MAP =
      objectMapper.readValue(METADATA, new TypeReference<>() {});
```